rocketbelt/components/tabcordions/rocketbelt.tabcordions.js

```diff
@@ -47,6 +47,13 @@
 
     // Click support
     $navlist.on('click', '.tabcordion_nav-item .tabcordion_nav-trigger', function () {
+      var currentTarget = $navlist.find('.tabcordion_nav-item.is-active .tabcordion_nav-trigger')[0];
+      if (currentTarget != $(this)[0]) {
+        var eventData = {'previousTarget': currentTarget, 'newTarget': $(this)[0]};
+        var event = new CustomEvent('tabcordion.TabChanged', {detail: eventData});
```

Comment:
Trying to come up with a preliminary convention for these—can you change the event name to `rb.tabcordion.tabChanged`?